samples/bluetooth/direct_test_mode/boards/nrf54lm20apdk_nrf54lm20a_cpuapp.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Disable the unsupported driver
+CONFIG_NRFX_TIMER0=n
+CONFIG_NRFX_TIMER1=n
+CONFIG_NRFX_TIMER2=n
+
+# Use necessary peripherals
+CONFIG_NRFX_TIMER20=y
```

Comment:
Is a timer 20 needed?

Reply:
Yes, it is needed here: https://github.com/nrfconnect/sdk-nrf/blob/045e3c32869221294053a669d2104a75b2807c97/samples/bluetooth/direct_test_mode/src/transport/dtm_uart_wait.c#L20

Reply:
Indeed, thanks for the clarification.